### PR TITLE
PCKPacker class example code

### PR DESCRIPTION
**Pull request by ghost**
It would be beneficial to add this simple piece of code to eliminate possible headaches that could occur from trying to use AddFile's method in an incorrect way without passing "res://".

### CODE REVIEW

File: doc/classes/PCKPacker.xml [hunk 1]
```diff
@@ -15,6 +15,14 @@
 			<argument index="1" name="source_path" type="String">
 			</argument>
 			<description>
+				The [PCKPacker] is used to create packages in application runtime.
+				[codeblock]
+				var Packer = new PCKPacker();
+				Packer.PckStart("Test.pck", 0)
+				Packer.AddFile("res://Text.txt", "Test.txt");
+				Packer.Flush(false);
```
Comment (YeldhamDev) [inline doc/classes/PCKPacker.xml]: The methods should be written in snake_case.

Comment (IlyaStuurFedorov) [reply doc/classes/PCKPacker.xml]: Sorry, I could not figure out how to make a tab-like-thing that allows switching in-between languages, I am going to port this code to gd then.

Comment (YeldhamDev) [reply doc/classes/PCKPacker.xml]: Language tabs aren't available in the internal docs.

**Review by YeldhamDev — APPROVED** [review]
Thanks for your first contribution!

File: doc/classes/PCKPacker.xml [hunk 2]
```diff
@@ -15,6 +15,14 @@
...
 			</argument>
 			<description>
+				The [PCKPacker] is used to create packages in application runtime.
+				[codeblock]
+				var packer = PCKPacker.new()
+				packer.pck_start("test.pck", 0)
+				packer.add_file("res://text.txt", "text.txt")
+				packer.flush(false)
+				[/codeblock]
+				The above [PCKPacker] creates package [b]test.pck[/b], then adds a file named [b]text.txt[/b] in the root of the package.
```
Comment (akien-mga) [inline doc/classes/PCKPacker.xml]: This sounds like this should go in the main `<description>` for the class itself, not for the `add_file` method.

The description for `add_file` should be more along the lines of "Adds the [code]source_path[/code] file to the current PCK package at the [code]pck_path[/code] internal path (should start with [code]res://[/code])."

**Comment by akien-mga** [comment]
See my above comment proposing some changes. Commits should eventually be squashed into one, see [PR workflow](https://docs.godotengine.org/en/3.1/community/contributing/pr_workflow.html#mastering-the-pr-workflow-the-rebase).

**Comment by IlyaStuurFedorov** [comment]
Oh god, something went terribly wrong on my end after rebasing

**Comment by akien-mga** [comment]
The final state looks good, but it looks like the rebase did not go as planned as there are 5 commits. I pushed a squashed commit directly as fa45fdb5f5d8435b49303ef36b33241f278b33de.

Thanks for your contribution!

**Pull request closed without merging**